resilience4j/assets/service_checks.json

```
@@ -0,0 +1,17 @@
+[
+    {
+        "agent_version": "7.59.0",
+        "integration": "Resilience4j",
+        "check": "resilience4j.prometheus.health",
```

```suggestion
        "check": "resilience4j.openmetrics.health",
```
nit: update the service_check name